scripts/ruby/lib/file_type.rb

```diff
@@ -96,6 +96,8 @@ def print_solr
     return transform_solr
   end
 
+  # these rules apply to XML to ES objects (DC / HTML / TEI / VRA)
```

Comment:
Two `to`s here read awkwardly. Is the first one a typo?

Reply:
No, it wasn't a typo, I mean the rules apply to a specific type of transformation for XML filetypes, but I could change the language.  "these rules apply to XML files transforming to ES" ?

Reply:
Ah okay. I suppose that might be clearer. Or `XML-to-ES objects` :)